src/Microsoft.VisualStudio.Services.Agent/SecretMask.cs

```diff
@@ -1,101 +0,0 @@
-using System;
-using System.Collections.Generic;
-using System.Text.RegularExpressions;
-
-namespace Microsoft.VisualStudio.Services.Agent
-{
-    public class ReplacementPosition
-    {
-        public ReplacementPosition(int start, int length)
-        {
-            Start = start;
-            Length = length;
-        }
-
-        public int Start { get; set; }
-        public int Length { get; set; }
-        public int End
-        {
-            get
-            {
-                return Start + Length;
-            }
-        }
-    }
-
-    public interface ISecret
```

Comment:
The class names changed, but the file was never renamed. I moved all of these classes into the existing file SecretMasker.cs, since they are all internal implementation details of the SecretMasker class.